app/views/admins/_stats.haml

```diff
@@ -0,0 +1,60 @@
+- content_for :head do
+  = stylesheet_link_tag :admin
+
+%h1= t(".usage_statistic")
+
+.row
+  = form_tag("/admins/stats", method: "get", class: "form-inline segment-selection") do
+    .col-sm-2.col-sm-push-8
+      %select.form-control{name: "range"}
+        %option{value: "daily", selected: ("selected" if params[:range] == "daily")}
```

Comment:
I'm quite sad to see you didn't apply the changes I proposed at https://github.com/diaspora/diaspora/pull/7295#discussion_r114014088 and https://github.com/diaspora/diaspora/pull/7295#discussion_r114016339. I know that this is a copy-pasted code, but I see no reasons to copy over lame code and to not introducing these little fixes that make that PR smaller, the code more readable and which are completely safe to make (it's just about code style).